src/PJ_wag3.c

```diff
@@ -30,6 +30,11 @@ static LP s_inverse (XY xy, PJ *P) {           /* Spheroidal, inverse */
 
 
 static void *freeup_new (PJ *P) {                       /* Destructor */
```

Comment:
I suggest using the simplified approach used in PJ_cart.c: Remove freeup_new(), and replace freeup() with the version from PJ_cart.c:
```
static void freeup (PJ *P) {
    pj_freeup_plain (P);
    return;
}
```

Later on, I expect to improve the PROJECTION macro, to replace freeup with pj_freeup_plain. It will, however require one pass over the code, cleaning up every PJ_*.c file (but it will result in removing most cases of freeup-functions, and a bit of init-code for each projection, at no cost at all (in the few cases where freeup does more advanced things than freeing P->opaque, then freeing P, we will have to add one line: P->pfree = freeup;)